browser/speedreader/speedreader_tab_helper.cc

```diff
@@ -393,6 +415,40 @@ void SpeedreaderTabHelper::OnDistillComplete(DistillationResult result) {
 #endif
 }
 
+void SpeedreaderTabHelper::OnReadingStart(content::WebContents* web_contents) {
+  constexpr const char16_t kReading[] = uR"js( setTtsReadingState($1) )js";
```

Comment:
is `js` a function in the global namespace? Given that the isolated world is shared by many brave scripts I think we should appropriately namespace things